transformer/common_data_provider.go

```diff
@@ -111,12 +117,38 @@ func (c CommonDataProvider) getNodeName() (string, error) {
 	return nName, nil
 }
 
+// FetchKubernetesVersion returns the version of the Kubernetes server
+func (c CommonDataProvider) FetchKubernetesVersion() (version.Version, error) {
+	serverVersion, err := c.kubeClient.Discovery().ServerVersion()
+	if err != nil {
+		return version.Version{}, err
+	}
+	return version.Version{
+		Version: serverVersion.Major + "." + serverVersion.Minor,
+	}, nil
+}
+
+func (c CommonDataProvider) FetchVersionInfo() (version.CloudbeatVersionInfo, error) {
+	serverVersion, err := c.FetchKubernetesVersion()
```

Comment:
Not a must, I would collect `Cloudbeat` & `Policy` versions and only then `k8s` to avoid calling the methods twice.